sematic/ui/src/index.tsx

```diff
@@ -173,11 +161,29 @@ function App() {
   );
 }
 
+function Router() {
```

Comment:
I upgraded `react-router-dom` to 6.8.1 to leverage the `loader` prop on the `<Route />` component. 

Another reason for updating is that what we currently use `react-router-dom@6.3`, is absent from officially supported documentation. `6.3` is an intermediate version that lacks consistent doc. The official doc only talks about the behaviors of 6.4+. Many features mentioned in the current doc do not work with 6.3.